src/asset-picker/AssetPickerDropDown.js

```diff
@@ -35,26 +35,27 @@ export default class AssetPickerDropDown extends Component {
             router.push('asset-picker');
         }
     }
+    onClose(e) {
+        const { actions } = this.props;
+        e.stopPropagation();
+        actions.updateAssetPickerSearchQuery('');
+        actions.updateAssetPickerFilter('all');
```

Comment:
if we add a new rule just to do the same thing. Wouldn't it be like doing the same thing with just a different name? I mean we have a rule that can just do that for us. The query is set my the same rule, modified and can as well clear by the same rule can't it?.